wormhole-connect/scripts/ofac/utils.ts

```diff
@@ -0,0 +1,14 @@
+export const getSanctionedAddresses = async (): Promise<string[]> => {
```

Comment:
Can we consolidate these files a little bit?  I think this could be moved into `getSdnList.ts` and exported from there